ark/phenotyping/cell_cluster_utils.py

```diff
@@ -477,17 +475,23 @@ def cluster_cells(base_dir, cell_pysom, cell_som_cluster_cols,
     if cell_pysom.weights is None:
         raise ValueError("Using untrained cell_pysom object, please invoke train_cell_som first")
 
+    # non-pixel cluster inputs won't be cell size normalized
```

Comment:
Where does size normalization happen? The generic data shouldn't automatically be size normalized

Reply:
This happens in `create_c2pc_data`, a step that is only run in `3_Pixie_Cluster_Cell`. This step is bypassed in `3b_Generic_Cluster_Cells` and thus the SOM inputs won't be size normalized for this.